src/Illuminate/Foundation/Vite.php

```diff
@@ -655,6 +655,24 @@ public function asset($asset, $buildDirectory = null)
         return $this->assetPath($buildDirectory.'/'.$chunk['file']);
     }
 
+    /**
+     * Get the content for an asset.
+     *
+     * @param  string  $asset
+     * @param  string|null  $buildDirectory
+     * @return string
+     */
+    public function content($asset, $buildDirectory = null)
+    {
+        $buildDirectory ??= $this->buildDirectory;
+
+        $chunk = $this->chunk($this->manifest($buildDirectory), $asset);
+
+        $path = public_path($buildDirectory.'/'.$chunk['file']);
+
+        return file_get_contents($path);
```

Comment:
I don't think we should do an unchecked call to file_get_contents.

Reply:
@GrahamCampbell added the check. Will it be satisfactory for most of the cases?